programs/ssl/ssl_server2.c

```diff
@@ -1759,7 +1759,7 @@ int report_cid_usage( mbedtls_ssl_context *ssl,
 
 int main( int argc, char *argv[] )
 {
-    int ret = 0, len, written, frags, exchanges_left;
+    int ret = 0, r = 0, len, written, frags, exchanges_left;
```

Comment:
Declare the variable just before to use it and name it 'psk_free_ret' ?
Like this the name of the variable does convey much information and the variable it used far away from where it is declared.

Reply:
Done